src/isWilsonPrime.js

```diff
@@ -19,7 +19,16 @@
  * @return {boolean}
  */
 function isWilsonPrime(num) {
-  // write code here
+  let factorial = 1;
+  for (let i = 1; i < num; i++) {
+    factorial *= num - i;
+  }
+
+  if (Number.isInteger((factorial + 1) / (num * num))) {
```

Comment:
there is no sense in this `if`

you can write simple
```suggestion
  return Number.isInteger((factorial + 1) / (num * num))
```
`Number.isInteger` - return Boolean value